draft-ietf-mls-protocol.md

```diff
@@ -1167,7 +1171,7 @@ object.
 
 [[ OPEN ISSUE: The Welcome message needs to be sent encrypted for
 the new member.  This should be done using the public key in the
-UserInitKey, either with ECIES or X3DH. ]]
+UserInitKey, either with HPKE or X3DH. ]]
```

Comment:
This OPEN ISSUE can be deleted.  The Welcome Message is encrypted as of -03; I just neglected to remove this comment.